src/modules/folder/folder.repository.ts

```diff
@@ -321,6 +321,17 @@ export class SequelizeFolderRepository implements FolderRepository {
     );
   }
 
+  async isOwner(user: User, folderId: number): Promise<boolean> {
```

Comment:
This is a find by id which apart from existing, contains the name of something that seems to be business (therefore, responsibility of the use case) related